src/dynamics/video_player/player/player.js

```diff
@@ -1074,15 +1074,22 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                                 return true;
                             }.bind(this),
```

Comment:
Hi @maxblessed, instead of creating `noFloatIfAboveCondition` what do you think of updating `floatCondition` to also validate `noFloatIfAbove` and `noFloatIfBelow`?

Reply:
@ntshcalleia . thanks for the feedback, I will update it straight away.